app/src/main/java/com/example/wordsapp/LetterAdapter.kt

```diff
@@ -41,9 +41,7 @@ class LetterAdapter :
         val button = view.findViewById<Button>(R.id.button_item)
     }
 
-    override fun getItemCount(): Int {
-        return list.size
-    }
+    override fun getItemCount(): Int = list.size
```

Comment:
I think it's possible to improve even more, what do you think about that?
```suggestion
    override fun getItemCount() = list.size
```